.github/workflows/rust.yml

```
@@ -25,23 +25,64 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      # Rust Tools are already installed and available in the Runner.
-      # See: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#rust-tools
       - if: contains(matrix.os, 'windows')
         # Rust & Cargo requires MSVC Build Tools on Windows.
         uses: seanmiddleditch/gha-setup-vsdevenv@v4
         with:
           arch: amd64
           host_arch: amd64
       - name: Build
+        # Rust Tools are already installed and available in the Runner.
+        # See: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#rust-tools
         run: cargo build --verbose --release
       - name: Run tests
         run: cargo test --verbose
         # This may fail, but it does not take effects to our artifacts.
         continue-on-error: true
-      - uses: actions/upload-artifact@v4
+      - name: Inspect latest version (Windows)
+        # This will run the release build as a normal user will do.
+        shell: bash -e {0}
+        run: |-
+          echo "::group::Create root directory for Minecraft"
+          mkdir -pv ./mc/
+          cd ./mc/
+          ../target/release/launcher.exe init
+          echo "::endgroup::"
+          echo "::group::Inspect the latest release number"
+          {
+            echo "MINECRAFT_VERSION" << EOF
```

```suggestion
            echo "MINECRAFT_VERSION << EOF"
```

See [documentation](https://docs.github.com/zh/actions/using-workflows/workflow-commands-for-github-actions#example-of-a-multiline-string)